Engines/Wine/Engine/Object/script.js

```diff
@@ -1,5 +1,5 @@
 const WineEngine = include("engines.wine.engine.implementation");
-const { fileExists } = include("utils.functions.filesystem.files");
+const { fileExists} = include("utils.functions.filesystem.files");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required before '}'.](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)